openstack/loadbalancer/v2/pools/requests.go

```diff
@@ -390,6 +411,21 @@ type BatchUpdateMemberOpts struct {
 	// The administrative state of the Pool. A valid value is true (UP)
 	// or false (DOWN).
 	AdminStateUp *bool `json:"admin_state_up,omitempty"`
+
+	// Is the member a backup? Backup members only receive traffic when all
+	// non-backup members are down.
+	// Requires microversion 2.1 or later.
+	Backup *bool `json:"backup,omitempty"`
+
+	// An alternate IP address used for health monitoring a backend member.
+	MonitorAddress string `json:"monitor_address,omitempty"`
```

Comment:
Is removing/unsetting a monitor address from a member possible? If so, is it done via an empty string (`""`) or with nil/null?

The combination of `string` and `omitempty` will cause values of empty strings to be removed from the request body. If an empty string is used to remove/unset the monitor address, this should be `*string`.

Reply:
Yes, it is possible. I tested it and for me

* setting it to `""`
* setting it to `null`
* omitting it

will remove/unset the monitor address. I'm unsure what the preferred way is.

Reply:
I recommend setting it to:

```go
MonitorAddress *string `json:"monitor_address,omitempty"`
```